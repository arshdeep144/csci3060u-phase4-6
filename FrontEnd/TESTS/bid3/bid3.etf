10 user01 BS 1000
04 item01 user02 user01 90
00 user01 BS 1000
