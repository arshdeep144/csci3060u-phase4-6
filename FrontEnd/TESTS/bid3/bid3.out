Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter the name of the item: Please enter the name of the seller: Please enter the number of days left on the item: Current bid is 55. How much would you like to bid?
Your new bid has been saved
Welcome to the main menu. Please enter a command to continue: 
